x-pack/plugins/siem/public/components/toasters/modal_all_errors.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiOverlayMask,
+  EuiModal,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiModalBody,
+  EuiCallOut,
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiModalFooter,
+  EuiAccordion,
+} from '@elastic/eui';
+import React from 'react';
+
+import { AppToast } from '.';
+import * as i18n from './translations';
+
+interface FullErrorProps {
+  isShowing: boolean;
+  toast: AppToast;
+  toggle: (toast: AppToast) => void;
+}
+
+export const ModalAllErrors = ({ isShowing, toast, toggle }: FullErrorProps) =>
+  isShowing && toast != null ? (
+    <EuiOverlayMask>
+      <EuiModal onClose={() => toggle(toast)}>
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>{i18n.TITLE_ERROR_MODAL}</EuiModalHeaderTitle>
+        </EuiModalHeader>
+
+        <EuiModalBody>
+          <EuiCallOut title={toast.title} color="danger" size="s" iconType="alert" />
```

Comment:
Looks like the ⚠️ icon isn't vertically centered:

![image](https://user-images.githubusercontent.com/2946766/59222669-47070900-8b87-11e9-8e03-6d231d98885d.png)

Reply:
May want matching bottom margin below the truncated/expandable message to give more buffer before the big grey area.
![image](https://user-images.githubusercontent.com/2946766/59222868-bed53380-8b87-11e9-8f84-ba2fe6fe9267.png)

Reply:
> Looks like the ⚠️ icon isn't vertically centered:
> 
> ![image](https://user-images.githubusercontent.com/2946766/59222669-47070900-8b87-11e9-8e03-6d231d98885d.png)

We should file this one under EUI repo